### PR TITLE
Module 2

### PR DESCRIPTION
**Pull request by SerhiiSmilianets**
added tests to created components in Module 1

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (MykhailoSheliahov) [inline cypress/support/commands.js:1]: ✂

File: src/tests/Counter.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import Counter from "../components/Counter";
+
+describe('Counter', () => {
+  it('renders initial value provided in props', () => {
```
Comment (MykhailoSheliahov) [inline src/tests/Counter.test.js:6]:
```suggestion
  it('should render initial value provided in props', () => {
```

**Review by MykhailoSheliahov — APPROVED** [review]
LGTM

File: cypress/support/commands.js (hunk 1 above)
Comment (SerhiiSmilianets) [reply cypress/support/commands.js:1]: @MykhailoSheliahov was added automatically with cypress installation

File: src/tests/Counter.test.js (hunk 2 above)
Comment (SerhiiSmilianets) [reply src/tests/Counter.test.js:6]: @MykhailoSheliahov fixed in Module 3

**Pull request merged**